sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/ParallelDocumentQueryExecutionContext.java

```diff
@@ -412,7 +412,7 @@ public EmptyPagesFilterTransformer(RequestChargeTracker tracker, CosmosQueryRequ
                 executeFunc,
                 targetRange,
                 collectionRid,
-                () -> client.getResetSessionTokenRetryPolicy().getRequestPolicy(),
```

Comment:
@mbhaskar  please check this, this is not related change , but found we were duplicating this code

Reply:
Good catch. This is redundant as the caller is doing the same.